lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -150,24 +159,21 @@ def which(cmd):
     LOGGER.warning("Could not find command '%s' (with permissions to read/execute it) in $PATH (%s)", cmd, paths)
     return None
 
-def check_output(time_passed, output_timeout):
+def loop_process_output_common(runmpiclass):
```

Comment:
create a separate subclass of `Run` for this, which adds this method, and then derive `RunFileLoopMPI` and `RunAsyncMPI` (also) from that subclass (so the method is available)